src/mmw/mmw/settings/gwlfe_settings.py

```diff
@@ -10,6 +10,7 @@
 GWLFE_DEFAULTS = {
     'NRur': 10,  # Number of Rural Land Use Categories
     'NUrb': 6,  # Number of Urban Land Use Categories
+    'NLU': 16, # Total Number of Land Use Categories
```

Comment:
Consider adding this to `GWLFE_CONFIG` instead, since this isn't a part of the actual DataModel which we initialize with `GWLFE_DEFAULTS`.